CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.67.0 / 2023-07-27
+
+* [FEATURE] Add `spec.scheme` field to the ScrapeConfig CRD. #5679
+* [FEATURE] Add `spec.params` field to the ScrapeConfig CRD. #5750
```

Comment:
we need to include https://github.com/prometheus-operator/prometheus-operator/pull/5742 as well :)